packages/sx.js/src/clients/offchain/types.ts

```diff
@@ -169,6 +169,7 @@ export type Propose = {
   body: string;
   discussion: string;
   choices: string[];
+  labels: string[];
```

Comment:
Don't we need to update those as well?
https://github.com/snapshot-labs/sx-monorepo/blob/7076db1633d69b7018f49f6a10e62ced9fb74a7a/packages/sx.js/src/clients/offchain/ethereum-sig/types.ts#L62-L78

https://github.com/snapshot-labs/sx-monorepo/blob/36bdb33f375f127a043cca574fe44fd1edf9c088/packages/sx.js/src/clients/offchain/types.ts#L52-L66